### PR TITLE
Add web-vitals measuring and event tracking

### PR DESCRIPTION
**Pull request by schamp**
See https://trello.com/c/CwE3us86/844-performance-add-reporting-of-fcp-to-google-analytics

### CODE REVIEW

**Comment by schamp** [comment]
image
 Looks like the events are getting sent to Analytics (from the test deployment on vercel)

**Review by pnavarrc — APPROVED** [review]
Looks good and works well, I just added a suggestion on naming the Event Actions so the reports look nicer in Google Analytics. 

Feel free to merge after fixing the linting warnings and renaming the event actions. Thanks for pushing this change!

File: src/index.js [hunk 1]
```diff
@@ -30,3 +32,5 @@ ReactDOM.render(<App />, document.getElementById('root'));
 // unregister() to register() below. Note this comes with some pitfalls.
 // Learn more about service workers: https://bit.ly/CRA-PWA
 serviceWorker.unregister();
+
+reportWebVitals(trackWebVitals);
```
[the remark this answers is not in the document]
Comment (schamp) [reply src/index.js:36]: I think that we should stick with the first load for now, I think we'll be in a better position to decide whether we want to add subsequent navigation once we've gotten used to interpreting the data.

File: src/components/Analytics/utils.ts [hunk 2]
```diff
@@ -65,6 +66,11 @@ export enum EventAction {
   REDIRECT = 'redirect',
   FOCUS = 'focus',
   NAVIGATE = 'navigate',
+  CLS = 'CLS', // Cumulative Layout Shift, for web-vitals
+  FCP = 'FCP', // First Contentful Paint, for web-vitals
+  FID = 'FID', // First Input Delay, for web-vitals
+  LCP = 'LCP', // Largest Contentful Paint, for web-vitals
+  TTFB = 'TTFB', // Time to First Byte, for web-vitals
```
[the remark this answers is not in the document]
Comment (schamp) [reply src/components/Analytics/utils.ts:73]: good idea, thanks!

**Pull request merged**